### PR TITLE
Add doc for statistics

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

File: presto-docs/src/main/sphinx/statistics/statistics.rst [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+or even one table layout and not the other for single table.
+
+Displaying table statistics
+---------------------------
+
+Table statistics can be displayed via Presto CLI using ``SHOW COLUMN STATS`` command.
+
+There are two flavors of the command:
+ * ``SHOW STATS FOR <table_name>`` - will show statistics for table layout representing all data in the table
+ * ``SHOW STATS FOR (SELECT <column_list|*> FROM <table_name> WHERE <filtering_condition>`` -
```
Comment (fiedukow) [inline presto-docs/src/main/sphinx/statistics/statistics.rst]: closing bracket missing

File: presto-docs/src/main/sphinx/statistics/statistics.rst [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+Depending which connector is in use only, subset of statistics may be available.
+For example Hive connector does not support data size for now.
+Also some statistics may be available for one table but not the other,
+or even one table layout and not the other for single table.
+
+Displaying table statistics
+---------------------------
+
+Table statistics can be displayed via Presto CLI using ``SHOW COLUMN STATS`` command.
```
Comment (kbajda) [inline presto-docs/src/main/sphinx/statistics/statistics.rst]: "COLUMN" is obsolete

File: presto-docs/src/main/sphinx/statistics/statistics.rst [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        ANALYZE TABLE t PARTITION (a=1, b) COMPUTE STATISTICS FOR COLUMNS``
+
+And this command will update stats for just one partition::
+
+        ANALYZE TABLE t PARTITION (a=1, b=5) COMPUTE STATISTICS FOR COLUMNS``
+
+For documentation of Hive statistics mechanism see https://cwiki.apache.org/confluence/display/Hive/StatsDev
+
+
+Cost in EXPLAIN
```
Comment (kbajda) [inline presto-docs/src/main/sphinx/statistics/statistics.rst]: I would make it a separate subsection.

**Review by kbajda — CHANGES_REQUESTED** [review]
Some minor comments.
How about moving this content under the "optimizer" doc section introduced in https://github.com/Teradata/presto/pull/424/ ?

File: presto-docs/src/main/sphinx/statistics/statistics.rst (hunk 3 above)
Comment (losipiuk) [reply presto-docs/src/main/sphinx/statistics/statistics.rst]: sure

File: presto-docs/src/main/sphinx/statistics/statistics.rst (hunk 1 above)
Comment (losipiuk) [reply presto-docs/src/main/sphinx/statistics/statistics.rst]: done

File: presto-docs/src/main/sphinx/statistics/statistics.rst (hunk 2 above)
Comment (losipiuk) [reply presto-docs/src/main/sphinx/statistics/statistics.rst]: done

**Comment by losipiuk** [comment]
> Some minor comments.
> How about moving this content under the "optimizer" doc section introduced in #424/ ?

Moved

**Comment by losipiuk** [comment]
addressed comments

**Comment by losipiuk** [comment]
@KBP-TDC please remove changes-requested tag if it is ok now.

**Review by kbajda — APPROVED** [review]

**Comment by losipiuk** [comment]
merged

**Pull request closed without merging**